contents/product/correlation-analysis.mdx

```diff
@@ -1,25 +1,25 @@
 ---
-title: Diagnose causes with Correlation Analysis
-featuredImage: ../images/product/quantitative-analysis.png
+title: Diagnose causes automatically with Correlation Analysis
+featuredImage: ../images/apps/quantitative-analysis.png
 ---
 
 <Section
     divider={false}
-    title="Correlation Analysis automatically suggests which events or user properties lead to success or failure using transparent, testable, statistical models."
+    title="Correlation Analysis suggests which events or properties lead to success or failure using transparent statistical models."
     size="full"
     cols={3}
 >
     <div>
-        <h5>Optimize funnels</h5>
-        <p>Understand what users were doing when they dropped-off, and which event(s) might be correlated.</p>
+        <h5>Works great with funnels</h5>
```

Comment:
Maybe something like "Understand your funnel"  or "Automated funnel insights" would be better?